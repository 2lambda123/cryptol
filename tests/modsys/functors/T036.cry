module T036 where

import T036_B

x = T { x = 0 }

