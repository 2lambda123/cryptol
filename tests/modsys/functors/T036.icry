:load T036.cry
x

