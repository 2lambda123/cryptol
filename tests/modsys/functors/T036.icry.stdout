Loading module Cryptol
Loading module Cryptol
Loading interface module `parameter` interface of T036_A
Loading module T036_A
Loading module T036_B
Loading module T036
Showing a specific instance of polymorphic result:
  * Using '0' for type argument 'n' of 'T036_B::T'
{x = 0x0}
